### DISCUSSION TITLE
Only 1 CPU-Core?

### DISCUSSION THREAD

**Discussion by ftothekey** [gotenberg/gotenberg#978] [Q&A] [answered]
Hello, I'm using Gotenberg 8 default docker container.

I'm using the LibreOffice convert route (/forms/libreoffice/convert). In my case I want to convert a .txt to a .pdf. 

It all works fine with small files. 

However with larger text files, e.g. 10MB it takes very long (which is to be expected) and I noticed that only 1 CPU-Core is being used by Gotenberg. Is this by design or a misconfig on my end?

I tested it with only 1 File at a time.

I'm hoping to decrease the processing time by utilizing multiple CPU-Cores.

![image](https://github.com/user-attachments/assets/738d0f9b-3b0d-4843-a15c-0ff9801540c8)

**Comment by gulien** [comment, marked as answer]
Hello @ftothekey 👋 

Gotenberg is a glorified wrapper around other tools, so in your case if LibreOffice does not take more than 1 CPU core, there is not much I can do alas 🫤 

Unless there is a known way to tell LibreOffice to use more CPU cores that I am not aware of 😄

**Reply by ftothekey** [reply]
Hi @gulien,

thanks for the info and quick reply! hmm ok, thats unfortunate...in this case my users just need a little more patience :D

Maybe this is a hint which could be added to the documentation :)
